test/specs/mainnet/transferringFunds/arbitrum.spec.js

```diff
@@ -1760,7 +1904,7 @@ describe('The PrimeSDK, when transfer a token with arbitrum network on the MainN
         // estimate transactions added to the batch
         // passing callGasLimit as 40000 to manually set it
         try {
-          await arbitrumMainNetSdk.estimate(null, null, 40000);
+          await arbitrumMainNetSdk.estimate({ callGasLimit: 4000 });
```

Comment:
I don't think 4000 is enough for any transaction so it will always error out saying insufficient gas limit should it not be a higher value like 40k?

Reply:
@vignesha22 modified the callgaslimit